src/duckling/canvas/tools/entity-move-tool.ts

```diff
@@ -99,15 +113,14 @@ export class EntityMoveTool extends BaseTool {
     }
 
     private _isSnapToGrid(event : CanvasMouseEvent) {
-        return event.shiftKey;
+        return !event.shiftKey;
     }
 
     private _adjustEntityPosition(adjustment : Vector) {
-        let oldPosition = this._entityPositionService.getPosition(this._selectionService.selection.value.entity);
-        this._entityPositionService.setPosition(
-            this._selectionService.selection.value.selectedEntity,
-            vectorAdd(oldPosition, adjustment),
-            this._mergeKey);
+        let entity = this._entitySystemService.getEntity(this._selectedEntityKey);
```

Comment:
This doesn't get called anymore. Entities can't be adjusted via the arrow keys anymore